src/firefly/js/visualize/ui/ColorDialog.jsx

```diff
@@ -96,35 +97,31 @@ function renderThreeColorView(plot, rFields, gFields, bFields, rgbFields, isHueP
         />;
 
     return (
-        <div>
+        <Box sx={{m:1}}>
             {threeColorStretchMode}
             {Boolean(isHuePreservingSelected) && renderHuePreservingThreeColorView(plot, rgbFields)}
             {!isHuePreservingSelected && renderStandardThreeColorView(plot, rFields, gFields, bFields)}
-        </div>
+        </Box>
     );
 }
 
 function renderHuePreservingThreeColorView(plot,rgbFields) {
     const groupKey = RGB_HUEPRESERVE_PANEL;
     return (
-        <div style={{paddingTop:4}}>
+        <Box sx={{m:1}}>
             <FieldGroup groupKey={groupKey} keepState={false} reducerFunc={rgbHuePreserveReducer} >
                 <ColorRGBHuePreservingPanel {...{plot, rgbFields, groupKey}}/>
-                <div style={{display: 'flex', alignItems: 'center', justifyContent: 'space-between'}}>
+                <Stack {...{direction: 'row', alignItems: 'center', justifyContent: 'space-between'}}>
                     <CompleteButton
-                        closeOnValid={false}
-                        style={{padding: '2px 0 7px 10px'}}
-                        onSuccess={(request)=>replot3ColorHuePreserving(request)}
-                        onFail={invalidMessage}
-                        text='Refresh'
-                        dialogId='ColorStretchDialog'
+                        closeOnValid={false} sx={{pt:.25, pb:1, pl:1}} text='Refresh' dialogId='ColorStretchDialog'
+                        onFail={invalidMessage} onSuccess={(request)=>replot3ColorHuePreserving(request)}
                     />
-                    <div style={{ textAlign:'right', padding: '2px 10px'}}>
+                    <Box sx={{ textAlign:'right', py:.25, px:1}}>
```

Comment:
I prefer `1/4` over decimal.  Of course, it's just a matter of preference.  
Also, I used fixed number of pixels in places where spacing should not change the appearance.  I don't know if this is the case here.